justfile

```diff
@@ -199,8 +200,14 @@ rust-clean:
     cargo clean
 
 # Cleans up darts build cache
+_dart-clean $WORKSPACE:
+    cd "$WORKSPACE"; \
+    find . -type d -name .dart_tool -prune -exec rm -r '{}' \;
```

Comment:
this exits with an error on macos if the `.` is not given as a path.

Reply:
@rustonaut can maybe you check if this still works for you?

Reply:
it should work as this is the same syntax used for example in `just remove-assets`